### ISSUE TITLE
scene manager

### ISSUE THREAD

**Issue by RichardMarks** [RichardMarks/coin-collector-mobile#16]
We need to split up the game into `scenes` so that we can keep things easy to maintain and to develop in parallel.

The scene manager is responsible for keeping track of the `current scene`, a list of `known scenes` and potentially (not in scope right now) a `next scene` to allow for a smooth animated transition between scenes.
